docs/components/input.html

```diff
@@ -118,7 +118,7 @@ <h4>Input field with helper text</h4>
                 <p>This is basic field with helper text in the form of suggestions to user.</p>
                 <div class="c-example">
                     <div class="input-group helper">
-                        <label>Username </label><br>
+                        <label>Username </label>
```

Comment:
You can use 'for' and 'id' attributes for a pair of label and input elements, if they are not nested.

Reply:
Since they might have different values as per need I have not included